CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,31 @@
+# Community Code of Conduct
+
+This code of conduct applies to all spaces managed by Columnar Technologies Inc. (hereafter “Columnar”). This includes issue trackers, pull requests, and any other forums created by Columnar which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```

Comment:
```suggestion
This code of conduct applies to all public online spaces managed by Columnar Technologies Inc. (hereafter “Columnar”). This includes issue trackers, pull requests, and any other forums created by Columnar which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```

Reply:
Conduct in _internal_ Columnar spaces is governed by Columnar's separate Code of Business Conduct and Ethics. Conduct in public / guests-invited _physical_ spaces managed by Columnar should probably be considered independently of this.